common/app/uiComponent/core/RenderingActor.scala

```diff
@@ -0,0 +1,23 @@
+package uiComponent.core
+
+import akka.actor.Actor
+import model.ApplicationContext
+import uiComponent.UIComponent
+import scala.util.Try
+
+case class Rendering(component: UIComponent, ac: ApplicationContext)
+
+case class RenderingException(error: String) extends RuntimeException(error)
+
+class RenderingActor extends Actor with JavascriptRendering {
+
+  override def javascriptFile: String = "ui.bundle.server.js"
```

Comment:
Let's not worry about it, I'm getting off topic anyway.  Nothing wrong with the PR, just keen to see how the react app comes together with this.